### ISSUE TITLE
Method Proposal: Soliciting Article Pitches

### ISSUE THREAD

**Issue by remixz** [hoodiehq/editorial#6]
As the interest in Hoodie and its editorial grows, I'm sure there'll be a lot of people who'll want to write something for the Hoodie blog. We talked a lot about recurring posts (e.g. TGIF, Meet the Hoodie, cute animal pics), but we should definitely also talk about handling one-offs. I think it's possible that if we leave it open enough, we could have a steady stream of interesting one-off pieces, and it might become sort of a series. (i.e. every X day of the week, a "guest" one-off post is published)

Here are my thoughts on how the process could go:

1. **Getting the pitch** - I think it makes sense that we keep pitches as open as possible, since we'd all want to give feedback on them. I think the easiest way to do this would be through GitHub issues on a repo (either this one, or a separate one).To make it easier for both the writer pitching an idea, and us as a team reviewing, we could offer a template for pitches. They'd just copy/paste the markdown into an issue, and fill in things like the gist of the article (~1 paragraph), the approximate length, their best guess on how long it would take to write, and anything else the writer wants to add. (To be clear, the writer can be anyone, whether already involved with our editorial team, or a complete newbie.)
2. **Reviewing** - After the writer has submitted their pitch, members of the editorial team should review it, and offer feedback. The feedback shouldn't necessarily be a yes/no (unless the writer submitted something wildly off-topic. we should document what sort of content we want), but just feedback for the writer to make an even stronger pitch. This feedback/review period should last a few days, to let as many of us as possible give feedback.
3. **Approving** - Once the feedback period is up, we then have to make the big decision: Do we want this article? As long as the feedback period went well, and the writer is still on board, I'd say that we would want it. To coordinate who does what, I think we can do a pretty simple system: If you feel strongly about the article, you can volunteer for a role for that article. I think the roles could be laid out as the writer, an editor (or multiple editors, depending on the article length/time availability), and "managers." The managers would be responsible for doing things such as:
  * figuring out when the article needs to be done by/when to publish
  * coordinating the social media posts for the article (likely writing them)
  * working with anyone on the technical side to get the post on hood.ie
  * organizing any additional volunteers, such as illustrators, photographers, etc.

  The reason I suggest multiple managers is that there are a lot of tasks there. If someone is able to put in the time commitment for all of the tasks, then they could volunteer themselves for that, but since we don't want someone to burn out, we should encourage people just to manage one portion.
4. **Writing** - Finally, writing! We should leave this up to the writer as much as possible, but we should encourage them to share openly in their process. Writing in a Google Doc would be nice, since we can see progress whenever, and help edit as they write, but as long as they at least share some a draft or two, I'd say that's good. During the writing process, the manager in charge of scheduling should check in with the writer to make sure they're doing well. We should encourage writers to join our Slack, for easy check-ins.
5. **Editing** - Once the writer has completed their final draft, they should post it to the issue (or just share a Google Doc, or whatever they prefer), so that the editor(s) can start making suggestions. If the editor is comfortable with it, the editor could convert the post to markdown as well, so we can start getting ready to post it on hood.ie. If not, the manager involved with the "technical" aspects should do it, or work with someone who can. Also, this is when we should start adding any illustrations or photographs to the article. Illustrators/photographers, or the manager in charge, should post their pictures on the issue. During this time, everyone helping with editorial can give feedback as well, even if they aren't an "editor."
6. **Posting** - The big event! At this point, the final copy should be done, and we should have a PR ready for the hood.ie site. The scheduling manager should have chosen a time to post the article, which is when the PR is merged. From there, we send out the social media posts, and let the writer bask in their new post. :smile: We should give major thanks to the writer, and encourage them to contribute again, whether it be another article, or any other role.

I think this structure works well, since we're delegating specific tasks to people, so they don't have to spending too much time on posts, and won't get burnt out. It also lets a lot of people be a part of an article, which gives us all something to do. The role that probably would have the biggest time commitment would be the scheduling manager, since they would have to work with multiple people to make sure everything is ready for the release of the article. However, with the help of the other managers, it shouldn't be too bad. Ultimately, it's something that'd have to be experimented with, and something we'd get better at the more we do it. I think we can adapt this process for our recurring series as well, mostly for the reviewing and managing parts.

... *whew!* That was a lot of words. These are, of course, just my ideas, so I'm very open to any feedback. Everything's better when we work on it together! :tada:

**Comment by bnb** [comment]
I think it would be better to keep everything in GitHub in terms of writing. Here is an alternative structure to the one that you proposed:

* Writing: Have the author fork this repostory. They can either clone it locally and write in their favorite text editor, or work from the GitHub interface. Either way, they'll use markdown. If the choose to work with the latter, there is a guide linked from the GitHub text editor that shows all Markdown formatting.

* Editing: If the writing method above is used, they can push their post to their fork and then send a PR. This also allows for line comments and automatic merges, which will greatly streamline the process of getting their posts edited and published.

Keeping everything in GitHub will be _extremely_ helpful for organization. There's not really a need to create an entire workflow that relies on multiple factors when there is a way to integrate it all centrally at GitHub.

**Comment by renrutnnej** [comment]
👍🏻 keeping it all in GitHub

-- 
Jenn Turner


On November 24, 2015 at 12:34:43 PM, &! (bitandbang) (notifications@github.com) wrote:

I think it would be better to keep everything in GitHub in terms of writing. Here is an alternative structure to the one that you proposed:

Writing: Have the author fork this repostory. They can either clone it locally and write in their favorite text editor, or work from the GitHub interface. Either way, they'll use markdown. If the choose to work with the latter, there is a guide linked from the GitHub text editor that shows all Markdown formatting.

Editing: If the writing method above is used, they can push their post to their fork and then send a PR. This also allows for line comments and automatic merges, which will greatly streamline the process of getting their posts edited and published.

Keeping everything in GitHub will be extremely helpful for organization. There's not really a need to create an entire workflow that relies on multiple factors when there is a way to integrate it all centrally at GitHub.

**Comment by janl** [comment]
>  I think it's possible that if we leave it open enough, we could have a steady stream of interesting one-off pieces, and it might become sort of a series. (i.e. every X day of the week, a "guest" one-off post is published)

I like this idea very much :)

I also like the rest of this, thank you for taking the time to write it all out. I’d add a step 0. where we write down guidelines for a pitch, with an example and everything, so that people know what we expect.

And I have one more addition, regarding the scheduling manager, maybe we can put some of the burden into a process. Say we agree that one-off posts are always published on Thursdays, and whenever a post is ready it is published on the next Thursday, if there are multiple ones, we post them in order on every consecutive Thursday. That way, we don’t need a single person to keep everything all together (when is the post ready, when is a good time to publish etc), but we just have a “publisher” that takes “final” posts (of any series), and posts them on the day in the schedule, IF there is one. If there is none in the queue, nothing happens, and the series takes a break.

This should work for most series except maybe “TGIF” and “The Monday Talk”, but we can handle those differently, if we have to. For now, we have no series with one planned, so lets solve that when we get there, no need to overthink.

tl;dr: I wouldn’t mind if we take this issue and put it into a PROCESS.md file, or something, next to README.md, that makes clear how we work.

👍

**Comment by remixz** [comment]
Thanks all for the feedback! I've created a pull request with your feedback, so we can make this official. https://github.com/hoodiehq/editorial/pull/12

**Comment by bnb** [comment]
I like this a lot. Just a nitpick: I would give the document titles shorter but descriptive names, and I would definitely remove spaces from the file names. For new file names, I would go with something like `pitch-template.md` and `blog-process.md`. 

We'll be setting a precedent that will become hard to change as time goes on, so we need  to get it right the first time. If anyone has a scheme you think might be good, please voice yourself!

**Comment by janl** [comment]
One more note on the “all on GitHub”, I think we should be open for people not using that workflow, but if possible we should guide them there.

**Comment by bnb** [comment]
Cleanup: Can this be closed, as https://github.com/hoodiehq/editorial/pull/12 has been merged?
